ansible/roles/test/files/ptftests/wr_arp.py

```diff
@@ -276,48 +286,65 @@ def tearDown(self):
         return
 
     def runTest(self):
-        print
         thr = threading.Thread(target=self.dut_thr, kwargs={'q_from': self.q_to_dut, 'q_to': self.q_from_dut})
         thr.setDaemon(True)
         thr.start()
 
-        uptime_before = self.req_dut('uptime')
-        if uptime_before.startswith('error'):
-            self.log("DUT returned error for first uptime request")
-            self.req_dut('quit')
-            self.assertTrue(False, "DUT returned error for first uptime request")
+        # test warm reboot arp basic
+        self.test_wrarp_basic()
 
-        self.records = defaultdict(dict)
-        self.stop_at = time.time() + self.how_long
+        sleep(5)
 
-        test_port_thr = threading.Thread(target=self.test_port_thr)
-        test_port_thr.setDaemon(True)
-        test_port_thr.start()
+        # test warm reboot arp, ferret/non-broadcast/vxlan
+        self.test_wrarp_advance()
```

Comment:
Where to install the everflow table for the arp pkt?

Reply:
it's auto setup during warm-reboot